### ISSUE TITLE
Adat Panchayath Mapping

### ISSUE THREAD

**Issue by manojkmohan** [osmkerala/osmkerala-tasks#38]
A primary discussion is completed with local volunteers for a mapping effort for adatt gramapanchayath. Its our 3rd effort to make lsg level micro maps for different purposes after Koorachundu (2014) and Veeloor (2016).  #MappingAdat

**Comment by manojkmohan** [comment]
Corrected the boundary of the panchayath with local knowledge and other resources https://www.openstreetmap.org/changeset/98140058
https://www.openstreetmap.org/changeset/98121758

**Comment by manojkmohan** [comment]
Adat local_authority admin_level -8 relation https://www.openstreetmap.org/relation/11340532

**Comment by manojkmohan** [comment]
Changesets Tracker https://osmcha.org/filters?aoi=2432a7de-7b48-4833-9fdf-01e60cba709a

**Comment by manojkmohan** [comment]
admin_level -10 Mapping Started

**Comment by manojkmohan** [comment]
Admin 10 Completed.

**Comment by manojkmohan** [comment]
A google sheet for tracking https://docs.google.com/spreadsheets/d/15h4g_ruUuP-oILr4fKk7H1dwtxkNlXH_Wm2BGTZ0Vng/edit#gid=0

**Comment by manojkmohan** [comment]
15/04/2021 
Available peoples meeting Jospeh Chittilappilli, Anirudhan Muthuvara, Dhanya, Hari, Indu.. Communicated with Panchayath Peoples and Other volunteers.  
![osm mapping adat meeting 2021-3](https://user-images.githubusercontent.com/452426/118887670-4ac96b80-b918-11eb-88ad-fb01f60a7a28.jpeg)
![osm mapping adat meeting 2021-2](https://user-images.githubusercontent.com/452426/118887672-4bfa9880-b918-11eb-833f-b9cbd7d4cbf0.jpeg)
![osm mapping adat meeting 2021-1](https://user-images.githubusercontent.com/452426/118887675-4c932f00-b918-11eb-864c-0f49ff923ef0.jpeg)

**Comment by manojkmohan** [comment]
![1](https://user-images.githubusercontent.com/452426/118889388-b01e5c00-b91a-11eb-96cd-c3d36be1a8c6.png)

**Comment by manojkmohan** [comment]
Building Footprint Mapping Task need to activate.

**Comment by manojkmohan** [comment]
Wetlands/ Ponds Mapping In Progress 
![4](https://user-images.githubusercontent.com/452426/119249313-1d4d1e00-bbb5-11eb-8e0c-6d2eb431f594.png)

**Comment by manojkmohan** [comment]
An online training section conducted May 23, 4pm (googlemeet) for introduction to OSM to ward level volunteers. Dhanya, Jospeh, Anirudhan & Vinayan attended.

**Comment by manojkmohan** [comment]
![Screenshot from 2021-05-25 21-08-15](https://user-images.githubusercontent.com/452426/119557566-fa418a80-bdbd-11eb-9596-0e0940552cec.png)
Covid Map 25 may 2021

**Comment by answerquest** [comment]
@manojkmohan created project in teachosm tasking manager: https://tasks.teachosm.org/projects/1314

**Comment by manojkmohan** [comment]
Thanks @answerquest

**Comment by manojkmohan** [comment]
#AdatMapping  OpenStreetMap (3rd Meeting)
2021 Sunday, May 30 · 4:00 – 5:30pm
Google Meet joining info
Video call link: https://meet.google.com/yhr-pewg-hts

**Comment by manojkmohan** [comment]
@answerquest The hash tag requested not coming in change sets #AdatMapping #MappingThrissur #OpenDataKerala 
Currently Manually added. Can you check in configuration.

**Comment by manojkmohan** [comment]
Wikipage created. need to move contents to there https://wiki.openstreetmap.org/wiki/Adat_Panchayat_Mapping

**Comment by answerquest** [comment]
@manojkmohan yes I'm unable to find the setting for that in the new interface. Please reply back on email, we needed to get together on a call to hammer things out.

**Comment by manojkmohan** [comment]
Thanks Nikhil. Following in email

**Comment by manojkmohan** [comment]
Mapping activity progressing slowly using hot tool
![Firefox_Screenshot_2021-06-02T20-40-44 159Z](https://user-images.githubusercontent.com/452426/120549225-ed054b00-c410-11eb-961b-716387e19825.png)

**Comment by manojkmohan** [comment]
Adat Mapping using OSM
Sunday, June 6 · 7:45 – 9:45pm
Google Meet joining info
Video call link: https://meet.google.com/uox-iopb-woc
Or dial: ‪(US) +1 620-399-6042‬ PIN: ‪951 257 698‬#
More phone numbers: https://tel.meet/uox-iopb-woc?pin=1954790504901

**Comment by manojkmohan** [comment]
For Mapping Waterways and Roads created another task in teachosm https://tasks.teachosm.org/projects/1318/

**Comment by manojkmohan** [comment]
data issues https://www.openstreetmap.org/changeset/107023230
